ledger/src/blockstore_db.rs

```diff
@@ -526,6 +541,21 @@ impl Rocks {
             Err(e) => Err(BlockstoreError::RocksDb(e)),
         }
     }
+
+    fn print_error_handling_instruction(rocks_error: &rocksdb::Error) {
+        let error_msg = format!("{:?}", rocks_error);
+        if error_msg.contains("Corruption:") {
+            error!(
```

Comment:
well, I don't think `error!` logs is enough because people won't see logs that carefully. I think we should gracefully terminate the validator process. so, i know it's a bit tedious, but i'd rather like to plumb the `exit` atomicbool way down to here from the validator startup process.